### PR TITLE
fix for new headlight faces map corner if player escapes from look_around

### PR DESCRIPTION
**Pull request by spacenookie**
look_around can return INT_MIN which results in the new headlight facing a map corner, which is almost never what the player wants. if this happens we substitute a sane value of 0 (straight ahead)

### CODE REVIEW

File: src/veh_interact.cpp [hunk 1]
```diff
@@ -2321,19 +2321,24 @@ void veh_interact::complete_vehicle()
...
+	      const double PI = 3.14159265358979f;
+	      dir = int(atan2(static_cast<float>(delta_y), static_cast<float>(delta_x)) * 180.0 / PI);
+	      dir -= veh->face.dir();
+	      while(dir < 0) {
+                  dir += 360;
+	      }
+	      while(dir > 360) {
+                  dir -= 360;
+	      }
+	    }
```
Comment (Coolthulhu) [inline src/veh_interact.cpp]: Indentations should be 4 spaces each.

**Comment by Coolthulhu** [comment]
Wouldn't it be better to keep asking the player until some meaningful result?

**Comment by Zireael07** [comment]
@Coolthulhu: IMHO if I escape from headlight direction menu, it means I don't want to bother. And I never found the direction selection intuitive, so 'straight ahead' being the default might actually be the best choice.

**Comment by Coolthulhu** [comment]
Works. Waiting for proper tabs (we should probably have warning on tabs in code)

**Pull request merged**